lib/Doctrine/ORM/Query/SqlWalker.php

```diff
@@ -891,7 +892,11 @@ public function walkRangeVariableDeclaration($rangeVariableDeclaration)
         );
 
         if ($class->isInheritanceTypeJoined()) {
```

Comment:
I doubt that aids readability since the decision is made by the caller (actually even that one's caller) and not the extracted function itself ... and it just modifies local state (i.e. `$sql`).